src/server/project.ts

```diff
@@ -1780,6 +1788,46 @@ namespace ts.server {
         getIncompleteCompletionsCache() {
             return this.projectService.getIncompleteCompletionsCache();
         }
+
+        /*@internal*/

```

Comment:
In project.setCompilerOptions you would need to update the options for this project as well?